analysers/analyser_merge_eclext_FR.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights François Lacombe - 2023                                    ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from modules.OsmoseTranslation import T_
+from dateutil.parser import parse
+from .Analyser_Merge import Analyser_Merge_Point, CSV, Load_XY, Conflate, Select, Mapping
+
+
+class Analyser_merge_Eclext_FR(Analyser_Merge_Point):
+    def __init__(self, config, source_url, dataset_name, source, srid, osmRef, logger = None):
+        Analyser_Merge_Point.__init__(self, config, logger)
+        self.def_class_missing_official(item = 8090, id = 11, level = 3, tags = ['merge', 'eclairage', 'eclext', 'fix:chair', 'fix:survey'],
+            title = T_('Street light not integrated'))
+        self.def_class_possible_merge(item = 8091, id = 13, level = 3, tags = ['merge', 'eclairage', 'eclext', 'fix:chair', 'fix:picture'],
+            title = T_('Street light integration suggestion'))
+        self.def_class_update_official(item = 8092, id = 14, level = 3, tags = ['merge', 'eclairage', 'eclext', 'fix:chair', 'fix:survey'],
```

Comment:
Ids should go to Analyser_Merge_Eclext_FR_Syane

Reply:
Is it necessary?

I was inspired by `analyser_merge_hydrants_FR.py`

Reply:
You split into two file to make reusable. So yes.

> I was inspired by analyser_merge_hydrants_FR.py

That bad. But in this case it works anyway.

Reply:
Should we create an issue for that?